deployments/helm/KubeArmor/templates/RBAC/roles.yaml

```diff
@@ -88,6 +88,16 @@ rules:
   - list
   - watch
   - update
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+  - watch 
+  - patch
+  - update   
```

Comment:
Correct me if I'm wrong, do we need patch and update at the node level?
AFAIK we are only changing labels/annotations on the node, right?
@rksharma95 @Aryan-sharma11

Reply:
> we are only changing labels/annotations on the node, right?

correct!